src/annotation/askui-annotation-step-reporter.ts

```diff
@@ -0,0 +1,80 @@
+import fs from "fs";
+import { Step, Reporter, Annotation, ReporterConfig } from "askui";
```

Comment:
Is `askui` already setup as a peer-dependency under package.json?

Reply:
?

Reply:
I do not understand the question to be honest 🤔  Using a AskUIReporter without askui installed as dependency makes no sense to me.

To answer the question: No it is not installed as dependency but as devDependency?
Shall I add it as a dependency to askui-reporters?

Reply:
it should be a peer dependency instead of a dev dependency to make clear where this is coming from --> the user has it in her/his/... environment (because as you said it makes not sense to have it without it)

Read more about it [here](https://nodejs.org/en/blog/npm/peer-dependencies).